package-lock.json

```diff
@@ -8236,7 +8236,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
+          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
Are these updates needed for the PR?

Reply:
Could you please revert these changes to package-lock.json? These happen from time to time, and we've never really figured out how to avoid them unfortunately.